homeassistant/components/vivotek/camera.py

```diff
@@ -77,19 +133,40 @@ def setup_platform(
     add_entities([VivotekCam(config, cam, stream_source)], True)
 
 
+async def async_test_config(
```

Comment:
I'd suggest you move this to `util.py` instead as it's shared in between and not only used here.